kobolds/biology.md

```diff
@@ -1,58 +1,66 @@
-## Physical Appearance
+# Kobold Biology
 
-### Anatomy
+1. [Anatomy](#anatomy)
+   1. [Basic Appearance](#basic-appearance)
+   1. [Scales & Coloration](#scales--coloration)
+1. [Physiology](#physiology)
+   1. [Reproduction](#reproduction)
+   1. [Gestation & Development](#gestation--development)
+
+
+## Anatomy
+
+### Basic Appearance
 
 Kobolds are a small species of people who are lizard- and bird-like. They are
-typically about 3 feet tall, with a variance of ±6 inches being common (though
+typically about 4 feet tall, with a variance of ±6 inches being common (though
```

Comment:
I don't know about this. I keep going back and forth. I probably need to find a 3-foot and a 4-foot object to stand next to to get a better idea of what feel right in the actual physical world.

Reply:
My 8yo is just over 4' 4" and that feels about right, so I'mma call 4±0.5 feet good.